dev-lang/lazarus/lazarus-2.2.4.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop
+
+# See https://wiki.freepascal.org/Lazarus_2.0_fixes_branch for notes and
+# changes in FPCVER. It *does* change between minor versions of lazarus.
+FPCVER="3.2.2"
+
+DESCRIPTION="Lazarus IDE is a feature rich visual programming environment emulating Delphi"
+HOMEPAGE="https://www.lazarus-ide.org/"
+SRC_URI="https://sourceforge.net/projects/${PN}/files/Lazarus%20Zip%20_%20GZip/Lazarus%20${PV}/${P}-0.tar.gz"
+
+LICENSE="GPL-2 LGPL-2.1-with-linking-exception"
+SLOT="0/2.2" # Note: Slotting Lazarus needs slotting fpc, see DEPEND.
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk2 +gui extras"
+REQUIRED_USE="gtk2? ( gui )"
+REQUIRED_USE="!gui? ( !gtk2 )"
+REQUIRED_USE="extras? ( gui )"
+
+DEPEND=">=dev-lang/fpc-${FPCVER}[source]
+	>=sys-devel/binutils-2.19.1-r1:=
+	gui? ( 
+	    !gtk2? ( dev-libs/libqt5pas:0/2.2 )
+	    gtk2? ( x11-libs/gtk+:2 )
+)"
```

Comment:
I'm not sure how much happy will be @mattst88 to see a package that requires gtk2

Reply:
I can very easily toss it out. It's left as an option right now, because we kinda like options. But I wholeheartedly support removing IUSE="gtk2". The whole point of the whole enterprise is to switch lazarus to qt5.

Reply:
Mine was a joke about it as gnome team is trying hard to take down gtk2 but actually I'm not the right person to decide for it. I'd suggest to ask to Matt or @thesamesam for more infos.